### PR TITLE
Disclaimer for background region

### PR DESCRIPTION
**Pull request by camipacifici**
### Description

Added a sentence to the "aperture photometry" plugin docs to tell the user that only standard regions (created with "replace") are acceptable as background regions. Please double check that the link I added takes you [here](https://jdaviz.readthedocs.io/en/latest/imviz/displayimages.html#defining-spatial-regions). I am not sure I got it right.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [x] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [x] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Did the CI pass? If not, are the failures related?
- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [x] Is a milestone set?
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: docs/imviz/plugins.rst
Comment (kecnry) [inline docs/imviz/plugins.rst:144]: This should fix the broken link/build:

```suggestion
    (see :ref:`imviz_defining_spatial_regions`).
```

**Review by kecnry — APPROVED** [review]
LGMT (see [rendered docs](https://jdaviz--1589.org.readthedocs.build/en/1589/imviz/plugins.html#simple-aperture-photometry))

**Review by pllim — APPROVED** [review]
LGTM too. Thanks!

Cami, if you have a JIRA ticket for this, pls update it accordingly after merge.

**Comment by pllim** [comment]
No need to wait for CI since this is only doc change. I will cancel CI and use admin override to merge.

**Comment by camipacifici** [comment]
There was no jira ticket, sorry.

**Pull request merged**
